safe-global/safe-wallet-web
# fix: add cancellation overview

## What it solves

Partially improves #2763

## How this PR fixes it

An overview of the cancellation process, as well as an info banner, has been added to the cancellation flow. All references to "skipping" a recovery attempt have been changed to "cancel".

## How to test it

Ensure a recovery attempt is queued and cancel it. Observe the new overview screen, as well as tweaked text on the "New transaction" screen.

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/585d4119-a62c-4558-a9f5-02accb5ac055)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/3a070d91-bd31-4bbb-9b50-be1e898d28e4)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/da9bc2d3-061c-4a4d-a425-224312d392fe)

## Checklist
